src/main/js/controls/Gantt/GanttConfig.js

```diff
@@ -111,6 +114,42 @@ const validateEventData = (content) => {
         throw new Error(errors.THROW_MSG_NO_DATA_POINTS);
     }
 };
+/**
+ * Checks if panning is enabled or not
+ *
+ * @private
+ * @param {object} config - config object used by the graph.
+ * @returns {boolean} returns true of panning enabled else false.
+ */
+export const isPanningModeEnabled = (config) => {
+    if (config.pan !== undefined && config.pan.enabled) {
+        return true;
+    }
+    return false;
+};
+/**
+ * Used to set the clamp and transition when panning is enabled or not.
+ *
+ * @private
+ * @param {object} config - config object used by the graph.
+ * @returns {undefined} returns nothing
+ */
+export const settingsDictionary = (config) =>
+    isPanningModeEnabled(config)
+        ? {
+              shouldClamp: false,
+              transition: {
+                  duration: 0,
+                  ease: "linear"
+              }
+          }
+        : {
+              shouldClamp: true,
```

Comment:
Do we want to have a property here to create dateline def? Only create the def and assign URL if this property is enabled?

Reply:
ok then what name should be given to that property like "createDatelineDef"

Reply:
How about `shouldCreateDatelineDefs: true/false`?

Reply:
ok I will add this property